bin/run-test.sh

```diff
@@ -2,6 +2,9 @@
 
 # Usage: -c to report coverage
 
+# Enable assertions
+export DEBUG_AGENT_ASSERTIONS=1
```

Comment:
We've used the structure `CLOUD_DEBUG_xxxx` for our other env vars. Might be good to keep things consistent.